changes/4583.fixed

```diff
@@ -0,0 +1 @@
+Update pinned version of social-auth-core to remove dependency on python-jose & it's dependency on ecdsa.
```

Comment:
Sorry - this should be a `.dependencies` rather than a `.fixed`, and also should be in past tense ("Updated pinned version...").